airbyte-db/lib/src/main/java/io/airbyte/db/DataTypeUtils.java

```diff
@@ -19,6 +19,7 @@
 
   public static final String DATE_FORMAT_PATTERN = "yyyy-MM-dd'T'HH:mm:ss'Z'";
   public static final DateFormat DATE_FORMAT = new SimpleDateFormat(DATE_FORMAT_PATTERN); // Quoted "Z" to indicate UTC, no timezone offset
+  public static final DateTimeFormatter ISO_TIME = DateTimeFormatter.ISO_TIME;
```

Comment:
what is the purpose for declaring variable that refers to the constant value?

Reply:
> kindly reminder that we also should bump the version for the source-mssql-strict-encrypt connector

Sure, will do that along with changelog updating after got all approvals right before merging

Reply:
There was another value at the beginning, just automatically replaced it at some moment.
Good catch, updated, thanks!